### PR TITLE
Handle if results of analyse_sql contains multiple messages

### PR DESCRIPTION
**Pull request by hacchi-san-dayo**
SSIA

### CODE REVIEW

File: sql/current_date_and_subquery_with_between_are_used.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+select
+  current_date,
+  column_a
+from
+  dataset.table
+where
+  _table_suffix between '2022-06-01'
+  and (
+    select dt from dates
+  )
```
Comment (hirosassa) [inline sql/current_date_and_subquery_with_between_are_used.sql]: Cloud you please add new line at the EOF?

**Review by hirosassa — COMMENTED** [review]
Thank you for your GREAT contribution! Added a comment.

**Comment by hirosassa** [comment]
And cloud you add the test about this change?

File: sql/current_date_and_subquery_with_between_are_used.sql (hunk 1 above)
Comment (hacchi-san-dayo) [reply sql/current_date_and_subquery_with_between_are_used.sql]: Thanks!
I fixed it in this commit
https://github.com/hirosassa/bqvalid/pull/16/commits/620010661546f167d617d935607add55805bc304

File: src/main.rs [hunk 2]
```diff
@@ -322,4 +328,29 @@ mod tests {
             assert!(!is_sql(&e));
         }
     }
+
+    #[test]
+    fn multiple_messages_in_single_sql_file() {
+        let mut parser = TsParser::new();
+        parser.set_language(language()).unwrap();
+
+        let sql = fs::read_to_string("./sql/current_date_and_subquery_with_between_are_used.sql").unwrap();
```
Comment (hirosassa) [inline src/main.rs]: Please apply `cargo fmt`
https://github.com/hirosassa/bqvalid/actions/runs/8936597183/job/24547590445?pr=16

**Review by hirosassa — APPROVED** [review]
Looks good! Thanks again for your first contribution!

**Pull request merged**